diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 0bad8cbd0c9bd067c5c46424dfe3db546d255691..1f52202fc9cfedc0c36a315cafc15b6383531996 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -1329,6 +1329,7 @@ if (!is_android && !is_fuchsia) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
