diff --git a/chrome/browser/ui/views/frame/browser_view.cc b/chrome/browser/ui/views/frame/browser_view.cc
index 1be95c104d0b2b7b84ff2d0b37a51133d121ffe4..59294a8fc2a40ca88f7df9dea14ca9b266e00e80 100644
--- a/chrome/browser/ui/views/frame/browser_view.cc
+++ b/chrome/browser/ui/views/frame/browser_view.cc
@@ -261,7 +261,7 @@
 #include "ui/views/win/scoped_fullscreen_visibility.h"
 
 // To avoid conflicts with the macro from the Windows SDK...
-#undef LoadAccelerators
+//#undef LoadAccelerators
 #endif
 
 #if BUILDFLAG(ENABLE_ONE_CLICK_SIGNIN)
