diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index f36e3862d5468e3c54bc9940afb6f48a2aaea42f..a3124830735faa42cb935f6f6887189da755af0b 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -1382,6 +1382,7 @@ if (!is_android && !is_fuchsia) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
