diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index db636e64f4b7bf63ac239dead4ee2cf28d99f82f..22490d2a590e8a4e7fb613db6c3d89d331815b61 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -1387,6 +1387,7 @@ if (!is_android && !is_fuchsia) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
