diff --git a/components/content_settings/core/browser/cookie_settings.cc b/components/content_settings/core/browser/cookie_settings.cc
index b1c017a14bbf260770cd55178aa2376b921d7e1f..7bfd4da646fe4ca8ed938e2a1faec746366c6274 100644
--- a/components/content_settings/core/browser/cookie_settings.cc
+++ b/components/content_settings/core/browser/cookie_settings.cc
@@ -191,6 +191,7 @@ ContentSetting CookieSettings::GetCookieSettingInternal(
   DCHECK(value);
   ContentSetting setting = ValueToContentSetting(value.get());
   bool block = block_third && is_third_party_request;
+  BRAVE_COOKIE_SETTINGS_GET_COOKIES_SETTINGS_INTERNAL
 
   if (!block) {
     FireStorageAccessHistogram(
