diff --git a/services/network/cookie_settings.cc b/services/network/cookie_settings.cc
index 5bf6d6e5601ea36d69fbb49caed394feeafbbf89..ddfddcd79ae9199a4e73c88e6bac5d2e42828c42 100644
--- a/services/network/cookie_settings.cc
+++ b/services/network/cookie_settings.cc
@@ -181,6 +181,7 @@ CookieSettings::GetCookieSettingWithMetadata(
     // Note: global settings are implemented as a catch-all (*, *) pattern.
     if (IsExplicitSetting(*entry))
       blocked_by_third_party_setting = false;
+    BRAVE_COOKIE_SETTINGS_GET_COOKIE_SETTINGS_INTERNAL
   }
 
   if (blocked_by_third_party_setting) {
